pkg/version/update.go

```diff
@@ -61,6 +61,10 @@ func CheckForUpdate(
 	}
 	q.Set("clientID", clientID)
 	q.Set("InstallationID", InstallationID)
+	if os.Getenv("BACALHAU_UPDATE_CHECKER_TEST") != "" {
+		q.Set("bacalhau_update_checker_test", "true")
+	}
+
```

Comment:
What will this do? Might be worth adding a comment here

Reply:
will do